File: SkaiLab.Invoice.Dal/Models/OrganisationInvoiceSetting.cs
using System;
using System.Collections.Generic;

namespace SkaiLab.Invoice.Dal.Models
{
    public partial class OrganisationInvoiceSetting
    {
        public string Id { get; set; }
        public string TermAndConditionForQuote { get; set; }
        public string TermAndConditionForInvoice { get; set; }
        public string TermAndConditionForPurchaseOrder { get; set; }

        public virtual Organisation IdNavigation { get; set; }
    }
}

File: SkaiLab.Invoice/ClientApp/src/app/models/organisation-invoice-setting.ts
export class OrganisationInvoiceSetting{
    id:string;
    termAndConditionForQuote:string;
    termAndConditionForInvoice:string;
    termAndConditionForPurchaseOrder:string;
    constructor(){
        this.id="";
        this.termAndConditionForInvoice="";
        this.termAndConditionForQuote="";
        this.termAndConditionForPurchaseOrder="";
    }
}